nnvm/include/nnvm/top/nn.h

```diff
@@ -550,6 +550,29 @@ struct L2NormalizeParam : public dmlc::Parameter<L2NormalizeParam> {
   }
 };
 
+
+struct ACCELParam : public dmlc::Parameter<ACCELParam> {
```

Comment:
TVM community is considering retiring nnvm in next release(0.7). Should we reconsider other options instead of adding to nnvm?

Reply:
We also support Relay so we don't depend on NNVM specifically. We started out exploring TVM with NNVM some time ago when there were still mostly NNVM tutorials around and it wasn't clear for some time how long it would still be in use so we added it for completeness to support both Relay and NNVM until this last one gets officially deprecated.